### PR TITLE
Add a login page to allow users to auth locally

### PR DESCRIPTION
**Pull request by lgarvey**
The login page allows users to authenticate against django, which is useful in the staging environment as the staging Core AD may be down.

Django-axes has been added with default configuration, which will lockout a user after 3 failed login attempts.  See django-axes documentation: https://django-axes.readthedocs.io/en/latest/ for more info.

NOTE: the login page is unstyled as it is not currently intended for public use.

This PR includes django migrations (from django-axes)

### CODE REVIEW

File: sso/localauth/templates/registration/login.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+{% if form.errors %}
+<p>Your username and password didn't match. Please try again.</p>
+{% endif %}
+
+{% if next %}
```
Comment (marcofucci) [inline sso/localauth/templates/registration/login.html]: What case is this block trying to cover?

File: sso/localauth/templates/registration/login.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    <td>{{ form.password.label_tag }}</td>
+    <td>{{ form.password }}</td>
+</tr>
+</table>
+
+<input type="submit" value="login" />
+<input type="hidden" name="next" value="{{ next }}" />
+</form>
+
+{% endblock %}
```
Comment (marcofucci) [inline sso/localauth/templates/registration/login.html]: newline missing ;)

File: sso/localauth/templates/registration/login.html (hunk 1 above)
Comment (lgarvey) [reply sso/localauth/templates/registration/login.html]: It was part of a stock login template. I can't see a use for it, so it has been removed!

File: sso/localauth/templates/registration/login.html (hunk 2 above)
Comment (lgarvey) [reply sso/localauth/templates/registration/login.html]: Fixed.

**Pull request merged**